components/tidb_query/src/expr/mod.rs

```diff
@@ -368,7 +368,7 @@ mod tests {
         let mut expr = Expr::default();
         expr.set_tp(ExprType::ScalarFunc);
         expr.set_sig(sig);
-        expr.set_field_type(FieldType::default());
+        expr.set_type(FieldType::default());
```

Comment:
Why is field_type renamed to type?

Reply:
This instance is a mistake (fixed in the updated commit). In most instances it should be correct because there is a field named `type` and rust-proto renames if to avoid clashing with the keyword.

Reply:
But in the protobuf actually it is indeed called `field_type`? See https://github.com/pingcap/tipb/blob/master/proto/expression.proto#L608 🤔

Reply:
Yes, sorry, that is what I meant. In this case it should be `set_field_type` and I fixed that in the latest commit. There are a couple of other cases where there is a field called just `type`.